scripts/docker-testnet/README.md

```diff
@@ -0,0 +1,10 @@
+# Setting up a local-testnet with Docker
+
+First and foremost, one needs to build the **seednode** & **node** images. Hence, the **_build.sh_**
+script is provided. This can be done, by invoking the script or building the images manually.
+
+```
+./build.sh  # (Optional) Can be ignored if you already have the images stored in the local registry.
+./start.sh  # Will start the local-testnet.
+./clean.sh  # Will stop and remove the containers related to the local-testnet.
```

Comment:
i think that a good flow here is to have also a `stop` step, in order to be able to have the following development flow:
- initially: `build`
- multiple times: `start` , `stop`
- and finally: `clean`

i think is not very good to have to clean everything on each run, something we just wan't to stop the squad and start it again, with the same/setup configuration, later on

Reply:
this is also related to the docker create comment

Reply:
This intended workflow adds some complexity. The `start.sh` uses the `docker run` command. If you stop all containers, they will only start again with`docker start` instead, otherwise it will throw an error. I don't see much of a benefit to this behaviour, as I need to persists all the `containerIds/containerNames` between stopping and restarting. One can achieve this manually by leveraging already available commands in the `clean.sh` script.